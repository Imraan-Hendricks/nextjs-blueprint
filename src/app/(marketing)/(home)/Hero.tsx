export function Hero() {
  return (
    <section>
      <h1 className='text-3xl'>Hero</h1>
    </section>
  );
}
