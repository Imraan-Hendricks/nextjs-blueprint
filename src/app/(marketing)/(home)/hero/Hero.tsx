export function Hero() {
  return (
    <section>
      <h1 className='text-8xl tracking-tighter'>Home</h1>
    </section>
  );
}
