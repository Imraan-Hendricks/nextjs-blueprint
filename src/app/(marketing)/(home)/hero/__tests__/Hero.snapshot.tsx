import { cleanup, render } from '@testing-library/react';
import { Hero } from '../Hero';

describe('Hero Snapshot Tests', () => {
  afterEach(() => {
    cleanup();
  });

  it('should match the snapshot', () => {
    const { container } = render(<Hero />);
    expect(container).toMatchSnapshot();
  });
});
