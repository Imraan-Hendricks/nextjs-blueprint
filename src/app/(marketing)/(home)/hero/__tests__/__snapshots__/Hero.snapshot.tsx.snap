// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Snapshot Tests should match the snapshot 1`] = `
<div>
  <section>
    <h1
      class="text-3xl"
    >
      Home
    </h1>
  </section>
</div>
`;
